gravitee-apim-console-webui/src/services-ngx/application-general-ng.service.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { HttpClient } from '@angular/common/http';
+import { Inject, Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { Constants } from '../entities/Constants';
+import { ApplicationDetails } from '../management/application/details/general/general-ng/applicationDetails';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ApplicationGeneralNgService {
```

Comment:
```suggestion
export class ApplicationService {
```
WDYT ?

Reply:
I was trying to do it, but we already have ApplicationService and it breaks

Reply:
that meen you should complete this service instead of create a new one